code/wdb/datastripe.q

```diff
@@ -68,6 +77,7 @@ upserttopartition:{[dir;tablename;keycol;enumdata;nextp]
     s:first raze value'[?[enumdata;();1b;enlist[keycol]!enlist keycol]];
 
     /- get process specific taildir location
+    basedir:` sv dir,.proc.procname;
```

Comment:
is `dir` here the same in `savetablesoverperiod` below? because there it looks like a similar line was replaced by `.wdb.hdbdir`, would the same be valid here?